src/BrazeKit-dev.js

```diff
@@ -709,6 +719,108 @@ var constructor = function () {
         }
     }
 
+    function prepareInitialConsent(user) {
+        var userConsentState = getUserConsentState(user);
+
+        var currentConsentPayload = generateBrazeConsentStatePayload(
+            userConsentState
+        );
+
+        latestUserBrazeConsentString = JSON.stringify(currentConsentPayload);
```

Comment:
Looking through `generateBrazeConsentStatePayload` I see that `currentConsentPayload` returns `{}` if , `JSON.stringify({})` will return `'{}``. I think this may throw off the subsequent `latestUserBrazeConsentString.length`. since `'{}'` has a length of 2.

I wonder if the better approach is to check if `!isEmpty(currentConsentPayload)`, then stringify.